examples/python/README.md

```diff
@@ -23,16 +21,30 @@ pip install onnx
 pip install onnxruntime
 ```
 
+Choose a model. Examples of supported ones are:
+- Phi-2
+- Mistral
+- Gemma 2B IT
+- LLama 7B
+
```

Comment:
please update the example folder below:

'''bash
cd examples/phi2/python
python -m onnxruntime_genai.models.builder -m microsoft/phi-2 -e cpu -p int4 -o ./example-models/phi2-int4-cpu
'''